core/controllers/resources.py

```diff
@@ -40,79 +40,40 @@ def get(self, generator_id):
             raise self.PageNotFoundException
 
 
-class ImageDevHandler(base.BaseHandler):
-    """Handles image retrievals (only in dev -- in production, image files are
-    served from GCS).
+class AssetDevHandler(base.BaseHandler):
+    """Handles image and audio retrievals (only in dev -- in production,
+    image and audio files are served from GCS).
     """
 
-    _IMAGE_PATH_PREFIX = 'image'
-
     @acl_decorators.open_access
-    def get(self, exploration_id, encoded_filename):
-        """Returns an image.
+    def get(self, exploration_id, asset_type, encoded_filename):
+        """Returns an asset file.
 
         Args:
             exploration_id: the id of the exploration.
```

Comment:
FYI, docstrings are missing typeinfo. Should be:

    exploration_id: str. The ID of the exploration.

and similarly below. Make sure to start the description with a capital letter and end it with a period.